cg/models/cg_config.py

```diff
@@ -633,6 +634,17 @@ def pdc_service(self) -> PdcService:
             self.pdc_service_ = service
         return service
 
+    @property
+    def run_directory_names_services(self) -> RunDirectoryNamesServices:
+        services = self.__dict__.get("run_directory_names_services_")
+        if services is None:
+            LOG.debug("Instantiating run directory names services")
+            services = RunDirectoryNamesServices(
+                pacbio=PacbioRunDirectoryNamesService(self.run_instruments.pacbio.data_dir)
```

Comment:
```suggestion
            pacbio_service = PacbioRunDirectoryNamesService(self.run_instruments.pacbio.data_dir)
            services = RunDirectoryNamesServices(pacbio=pacbio_service)
```

Reply:
For me both are basically the same, but the original looks more expandable (only adding one line instead of two), I will keep it the way it is

Reply:
Do so!